docs/contributing/documentation-guidelines/index.md

```diff
@@ -19,3 +21,36 @@ In terms of style, you should refer to the [Google developer documentation style
 - [Use sentence case](https://developers.google.com/style/capitalization) for document titles and section headings.
 - [Use descriptive link text](https://developers.google.com/style/link-text).
 - [Write short sentences](https://developers.google.com/style/translation#write-short,-clear,-and-precise-sentences) that are easy to understand and translate.
+
+## How to preview your modification on Autoware Documentation
+
+There are two ways to preview your modification on Autoware Documentation.
+
+### 1. Use Github Actions workflow in `autowarefoundation/autoware-documentation` repository
+
+You can use Github Actions in `autowarefoundation/autoware-documentation` repository to deploy your branch.
```

Comment:
I feel this sentence can be simplified because it looks like just repeating the previous information.

```suggestion
Follow the steps below.
```

Or I guess you can even drop this sentence.
```suggestion
```

Reply:
Resolved https://github.com/autowarefoundation/autoware-documentation/pull/203/commits/ff49a125b82682e0d062b355a2a4a3258451dcd9